build.gradle

```diff
@@ -22,14 +21,6 @@ configurations {
     agentDeps.transitive = false
 }
 
-dependencyCheck {
```

Comment:
see this has all been taken out - is it only circle that can therefore run the snyk job or can it be run locally first?  The feedback cycle would be quite slow if can only be run in circle...

Reply:
hey peter, you can check locally by running `snyk test` in the root project dir.